alacritty/src/config/font.rs

```diff
@@ -22,6 +22,8 @@ pub struct Font {
     /// Glyph offset within character cell.
     pub glyph_offset: Delta<i8>,
 
+    #[config(removed = "to enable thin strokes, run:\n$ defaults write org.alacritty \
+                        AppleFontSmoothing -int 0")]
```

Comment:
This seems a bit verbose and I feel like it probably doesn't look great in the message bar. Should probably shorten it and remove the suggested command?

Reply:
It looks like that https://github.com/alacritty/alacritty/pull/6186#discussion_r917448769 I do think that it's long, but I don't care that much.

Reply:
I mean you can't copy from the message bar anyway, so what's the point? Just point users to configure it in macOS or omit it entirely.

Reply:
Yeah, I was suggesting something like `to control font thin strokes use AppleFontSmoothing user default`.

Reply:
We had a lot of back and forth on this one. I think it looks decent (screencap in comment), but I will change it to gesture vaguely at `AppleFontSmoothing` instead if that's preferred. https://github.com/alacritty/alacritty/pull/6186#discussion_r917448769

My main motivation in showing a command is that just mentioning the name of the key and the phrase "user default" is probably not sufficient information for many people learn how to enable this, and you might continue to get issue spam about it. macOS does not expose this setting in its UI anymore; you basically have to google for it. On top of that you need to know that it needs to be set specifically to 0 as an integer for it to work.

Reply:
Here is a shorter one (pushed just now).
<img width="802" alt="shorter_removed_message" src="https://user-images.githubusercontent.com/2592855/178391926-1bb30a8b-74ca-44df-b6a1-f4e03f520e87.png">

Reply:
Just link back to this thread when someone files an issue complaining that `AppleFontSmoothing` isn't a supported key in `alacritty.yml` after reading this. 😄

Reply:
Current state as of 18020cd:
<img width="802" alt="was_removed" src="https://user-images.githubusercontent.com/2592855/178397647-17a3acaf-9980-4707-b0b6-1643e8fed23e.png">